app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/individual/BusinessNameConfirmation.scala.html

```diff
@@ -30,16 +32,26 @@
     govukSummaryList : GovukSummaryList
 )
 
-@(confirmationForm: Form[YesNo], postAction: Call, backUrl: String, name: String)(implicit request: Request[_], messages: Messages)
+@(confirmationForm: Form[YesNo], postAction: Call, backUrl: String, name: String, isBusinessName:Boolean)(implicit request: Request[_], messages: Messages)
 
-@mainTemplate(title = messages("individual.business-name-confirmation.heading"), optForm = Some(confirmationForm), backLink = Some(backUrl)) {
+@mainTemplate(title = messages(headingKey), optForm = Some(confirmationForm), backLink = Some(backUrl)) {
+
+        @if(!isBusinessName){
+            @hmrcPageHeading(PageHeading(
+                    text = messages(headingKey),
+                    section = Some(messages("individual.business-name-confirmation.heading.caption")),
+                    headingClasses = Some("govuk-heading-l"),
+                    captionClasses = Some("govuk-caption-l")
+            ))
+        } else {
+            @hmrcPageHeading(PageHeading(
+                    text = messages(headingKey),
+                    section = Some(messages("individual.business-name-confirmation.heading.caption")),
+                    headingClasses = Some("govuk-heading-l"),
+                    captionClasses = Some("govuk-caption-l")
+                ))
+        }
```

Comment:
Both the if and the else are the same, the difference is already handled by the headingKey
You can remove the if and one of it's cases